src/mlpack/methods/ann/activation_functions/isru_function.hpp

```diff
@@ -0,0 +1,130 @@
+/**
+ * @file isru_function.hpp
+ * @author Prince Gupta
+ *
+ * Definition and implementation of the ISRU function
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+ 
+ #ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_ISRU_FUNCTION_HPP
+ #define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_ISRU_FUNCTION_HPP
+ 
+ #include <mlpack/prereqs.hpp>
+ 
+namespace mlpack{
+namespace ann /** Artifical Neural Network. */{
+  
+/**
+ * The ISRU function, defined by
+ * 
+ * @f{eqnarray*}{
+ * f(x) &=& x/sqrt(1 + alpha*x*x)
+ * f'(x) &=& cube(1/sqrt(1 + alpha*x*x))
+ * @f
+ */
+
+class ISRUFunction
+{
+  
+ public:
+  /**
+   * Computes the ISRU function
+   * 
+   * @param x Input data.
+   * @param alpha paramter, default value = 0.1
+   * @return f(x)
+   */
+  static double Fn(const double x, const double alpha = 0.1)
+  {
+    return x / (std::sqrt(1 + alpha*x*x));
+  }
+  
+  /**
+   * Computes the ISRU function using a matrix as a input
+   * 
+   * @param x Input data.
+   * @param y The resulting output activation
+   * @param alpha parameter, default value = 0.1
+   */
+  template<typename eT>
+  static void Fn(const arma::Mat<eT>& x, arma::Mat<eT>& y, const double alpha = 0.1)
+  {
+    y = x / (arma::sqrt(1 + alpha * arma::pow(x,2)));
```

Comment:
Add space after comma

Reply:
Thanks for pointing out all style errors. I have fixed them.